en/audio-system/overview.md

```diff
@@ -0,0 +1,22 @@
+# Overview of sound systems
+
+The interface of the sound system is mainly for two types of requirements, one category is "music" that is longer in length and is used for loop play, the other category is "sound effect" that is shorter in length and is used one-time play. <br>
```

Comment:
Original:
```
The interface of the sound system is mainly for two types of requirements, one category is "music" that is longer in length and is used for loop play, the other category is "sound effect" that is shorter in length and is used one-time play. <br>
```

Better:
```
There are __two__ types of sounds available in the audio system: __sound effects__ and __music__.  __Sound effects__ are short bursts of quick sounds that signal the player of the game making progress. A few examples of __sound effects__ are __gun noises__, __bullets firing__, a __character jumping__, __physics contact events__ and many others. __Music__ is longer in length and usually played in a loop. A few examples of __music__ are __background music__, __cut scenes__, __successfully completing a milestone in the game__ and many others.

The interface of the sound system is mainly for two types of requirements, one category is "music" that is longer in length and is used for loop play, the other category is "sound effect" that is shorter in length and is used one-time play. <br>
```

Reply:
I think we should follow the original meaning of Chinese.

Reply:
It's not the bible, I agree that @slackmoehrle version is more nature and explains more concretely how the system should be using in the real scenarios

Reply:
Yes, I also like it more. Let me correction it.